discord/commands/commands.py

```diff
@@ -527,17 +527,24 @@ async def _invoke(self, ctx: ApplicationContext) -> None:
         else:
             await self.callback(ctx, **kwargs)
 
-    async def invoke_autocomplete_callback(self, interaction: Interaction):
-        values = { i.name: i.default for i in self.options }
+    async def invoke_autocomplete_callback(
+            self,
+            interaction: Interaction,
+            cls: Optional[Type[AutocompleteContext]] = None
+    ):
+        if cls is None:
+            cls = AutocompleteContext
```

Comment:
```suggestion
            cls: AutocompleteContext = AutocompleteContext
    ):
```

Reply:
Think that's wrong. Should be `Type[AutocompleteContext]`